layouts/_default/single.html

```diff
@@ -15,7 +15,7 @@
       <ul>
         <li>{{ $location.street }} {{ $location.housenumber }} in {{ $location.postcode }} {{ $location.city }}</li>
         {{ with $location.opening_hours }}<li>Öffnungszeiten: {{ . }}</li>{{ end }}
-        {{ with $location.phone }}<li>Telefon: {{ . }}</li>{{ end }}
+        {{ with $location.phone }}<li>Telefon: <a href="tel:+01-562-867-5309">{{ . }}</a></li>{{ end }}
```

Comment:
It's not intended to have the example telephone number hard coded in source code.

You have to find a way to manipulate `$location.phone` string to match the example scheme. Have a look at Hugo Functions https://gohugo.io/functions/

Reply:
what does $location.phone string look like?

Reply:
It's an international number with spaces, for example `+49 421 69 69 77 67` via https://lokal.ninja/bremen/bremen/albers-maritim/

Reply:
oh alright. 

Any idea as to why I'm seeing these error?
![error](https://user-images.githubusercontent.com/40003718/95338795-ec8fd100-08d0-11eb-9e4d-8d6b7b9aea9b.JPG)

Reply:
Yes, telephone number with hyphens.

Where do you got an error?

Reply:
I installed hugo on my windows machine and followed the instructions on the ReadMe file. The errors above appear when I attempt to execute command "hugo server".

Reply:
```suggestion
      {{ with $location.phone }}<li>Telefon: <a href="tel:{{ replace $location.phone ' ' '-'}}">{{ . }}</a></li>{{ end }}
```
Meanwhile, the above line is what I have in mind with regards to the manipulation required. Let me know if this is right so that I can create a fresh PR.

Reply:
Sorry, I cannot see your errors you are getting after `hugo server`.
The `replace` is looking good. Although you can try to use `.` instead of `$location.phone` inside the function.
No need to create another PR. Just push another commit to your branch of this repo.